### PR TITLE
Create yaml file to clone and run onboarding e2e tests

### PR DESCRIPTION
**Pull request by pjarugula**
PR #1005 raised again via this repo (the original PR was raised from a fork)

### CODE REVIEW

**Comment by abasha1234** [comment]
ok gc

**Comment by abasha1234** [comment]
gc

**Comment by abasha1234** [comment]
gc build

File: test/deploy/onboarding-tests/roles/configure/tasks/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (NSSPKrishna) [inline test/deploy/onboarding-tests/roles/configure/tasks/package-lock.json:1]: Could you please provide some context on where this is being utilised in the code?

**Review by NSSPKrishna — APPROVED** [review]

**Pull request merged**